packages/wonder-blocks-icon-button/components/icon-button-core.js

```diff
@@ -192,6 +192,7 @@ const _generateStyles = (color, kind, light) => {
         disabled: {
             color: light ? mix(fade(white, 0.32), color) : offBlack32,
             cursor: "default",
+            pointerEvents: "none",
```

Comment:
Given that the designs are supposed to put the IconButtons so close together I think we should probably just not fix this since I think we'll want tooltips on disabled items including IconButtons.